[2025-04-28T00-46-48-876Z] ERROR:
Name: Error
Message: UNHANDLED_REJECTION
Stack:
Error: UNHANDLED_REJECTION
    at Object.error (/home/container/events/logger.js:152:27)
    at process.<anonymous> (/home/container/index.js:210:10)
    at process.emit (node:events:518:28)
    at emitUnhandledRejection (node:internal/process/promises:252:13)
    at throwUnhandledRejectionsMode (node:internal/process/promises:388:19)
    at processPromiseRejections (node:internal/process/promises:475:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:106:32)
Metadata: {
  "requestBody": {
    "files": [],
    "json": {
      "type": 4,
      "data": {
        "content": "Error al procesar la interacci√≥n",
        "tts": false,
        "enforce_nonce": false,
        "flags": 64
      }
    }
  },
  "rawError": {
    "message": "Unknown interaction",
    "code": 10062
  },
  "code": 10062,
  "status": 404,
  "method": "POST",
  "url": "https://discord.com/api/v10/interactions/1366214033421897868/aW50ZXJhY3Rpb246MTM2NjIxNDAzMzQyMTg5Nzg2ODoxaXp3cWNlQUM5eGRTclpGOGdVcmlSdGo2YVBVQXg3dGVFbmZ3ZjlZYm5rQmZDVEt3S2xNdXRkY3dldk5VRXZHMGlGVGJQd2xVWGtreUFkM2hQenNIUXZFelNBSXpvSW9hNU01NmF1SG5NSVROT2pSSlA2QVR1S0JwSHhYMUVBcQ/callback?with_response=false"
}

